lib/AzureClient/index.js

```diff
@@ -15,8 +15,9 @@ const GIT_REMOTE_NAME = "azure";
 
 module.exports = class AzureClient {
     addLogpoint(file, line, expression, linux = false) {
+        const { escape } = require("querystring");
         const command = linux ? `node webSSHClient.js ${file} ${line} ${expression}` :
-                                `curl -s "http://localhost:8967/logpoint?file=${file}&line=${line}&expression=${expression}"`;
+                                `curl -s "http://localhost:8967/add?file=${file}&line=${line}&expression=${escape(expression)}"`;
```

Comment:
This change is mostly to fix a bug where the logpoint expression needs to be URL encoded (e.g. if your expression is `2+2`, the `+` needs to be escaped).